### PR TITLE
Update the CRI documentation for 1.7

### PR DESCRIPTION
**Pull request by yujuhong**

### CODE REVIEW

File: contributors/devel/container-runtime-interface.md [hunk 1]
```diff
@@ -35,44 +35,49 @@ For Kubernetes 1.6:
...
   - [Exec/attach/port-forward streaming requests](https://docs.google.com/document/d/1OE_QoInPlVCK9rMAx9aybRmgFiVjHpJCHI9LrfdNM_s/edit?usp=sharing)
   - [Container stdout/stderr logs](https://github.com/kubernetes/kubernetes/blob/release-1.5/docs/proposals/kubelet-cri-logging.md)
-  - [Networking](https://github.com/kubernetes/community/blob/master/contributors/devel/kubelet-cri-networking.md)
 
 ## Work-In-Progress CRI runtimes
 
  - [cri-o](https://github.com/kubernetes-incubator/cri-o)
  - [rktlet](https://github.com/kubernetes-incubator/rktlet)
  - [frakti](https://github.com/kubernetes/frakti)
+ - [cri-containerd](https://github.com/kubernetes-incubator/cri-containerd)
```
Comment (Random-Liu) [inline contributors/devel/container-runtime-interface.md]: Thank!

File: contributors/devel/container-runtime-interface.md [hunk 2]
```diff
@@ -35,44 +35,49 @@ For Kubernetes 1.6:
...
-We plan to add CRI specifications/requirements in the near future. For now,
-these proposals and design docs are the best sources to understand CRI
-besides discussions on Github issues.
+
+Below is a mixed list of CRI specifications/requirements, design docs and
+proposals. We are working on adding more documentation for the API.
 
   - [Original proposal](https://github.com/kubernetes/kubernetes/blob/release-1.5/docs/proposals/container-runtime-interface-v1.md)
+  - [Networking](https://github.com/kubernetes/community/blob/master/contributors/devel/kubelet-cri-networking.md)
+  - [Container metrics](https://github.com/kubernetes/community/blob/master/contributors/devel/cri-container-stats.md)
```
Comment (Random-Liu) [inline contributors/devel/container-runtime-interface.md]: ? Deadlink.

File: contributors/devel/container-runtime-interface.md [hunk 3]
```diff
@@ -35,44 +35,49 @@ For Kubernetes 1.6:
      `--container-runtime-endpoint` and `--image-service-endpoint`.
    - Use the "remote" runtime by `--container-runtime=remote`.
 
-Please see the [Status Update](#status-update) section for known issues for
-each release. Note that CRI API is still in its early stages. We are actively
-incorporating feedback from early developers to improve the API. Developers
-should expect occasional API breaking changes.
+CRI is still young and we are actively incorporating feedback from  developers
```
Comment (Random-Liu) [inline contributors/devel/container-runtime-interface.md]: double spaces before "developers"

File: contributors/devel/container-runtime-interface.md [hunk 4]
```diff
@@ -35,44 +35,49 @@ For Kubernetes 1.6:
...
 ## Work-In-Progress CRI runtimes
 
  - [cri-o](https://github.com/kubernetes-incubator/cri-o)
  - [rktlet](https://github.com/kubernetes-incubator/rktlet)
  - [frakti](https://github.com/kubernetes/frakti)
+ - [cri-containerd](https://github.com/kubernetes-incubator/cri-containerd)
 
 ## [Status update](#status-update)
+### Kubernetes v1.7 release (Docker-CRI integration GA, container metrics API)
+  - The Docker CRI integration has been promoted to GA.
```
Comment (Random-Liu) [inline contributors/devel/container-runtime-interface.md]: Do we need to mention `--enable-cri` flag is removed?

File: contributors/devel/container-runtime-interface.md (hunk 2 above)
Comment (yujuhong) [inline contributors/devel/container-runtime-interface.md]: It's being added in another PR #742

File: contributors/devel/container-runtime-interface.md (hunk 4 above)
Comment (yujuhong) [inline contributors/devel/container-runtime-interface.md]: Will do

File: contributors/devel/container-runtime-interface.md (hunk 3 above)
Comment (yujuhong) [inline contributors/devel/container-runtime-interface.md]: done

File: contributors/devel/container-runtime-interface.md (hunk 2 above)
Comment (Random-Liu) [inline contributors/devel/container-runtime-interface.md]: ACK

**Comment by yujuhong** [comment]
Addressed the comments.

**Comment by Random-Liu** [comment]
LGTM

**Pull request merged**